javascript/src/html_processor.ts

```diff
@@ -277,7 +282,50 @@ class Paragraph {
   get text(): string {
     return this.nodes.map(node => node.text).join('');
   }
+
+  get lastNode(): NodeOrText | undefined {
+    return this.nodes.length ? this.nodes[this.nodes.length - 1] : undefined;
+  }
+  setHasBreakOpportunityAfter() {
+    if (this.nodes.length) {
+      this.lastNode!.has_break_opportunity_after = true;
+    }
+  }
+
+  /**
+   * @returns Indices of forced break opportunities in the source.
+   * They can be created by `<wbr>` tag or `&ZeroWidthSpace;`.
+   */
+  forcedOpportunities(): number[] {
```

Comment:
Can we consider transforming this into a getter or changing the function name to something like `getForcedOpportuniries`?

Reply:
Done. Chose `get` function as this isn't super light.